includes/data-stores/class-wc-admin-notes-data-store.php

```diff
@@ -199,10 +199,50 @@ private function save_actions( &$note ) {
 	/**
 	 * Return an ordered list of notes.
 	 *
+	 * @param array $args Query arguments.
 	 * @return array An array of objects containing a note id.
 	 */
-	public function get_notes() {
+	public function get_notes( $args = array() ) {
 		global $wpdb;
-		return $wpdb->get_results( "SELECT note_id, title, content FROM {$wpdb->prefix}woocommerce_admin_notes order by note_id ASC;" );
+
+		// Build the query.
+		$query = "
+			SELECT note_id, title, content
+			FROM {$wpdb->prefix}woocommerce_admin_notes
+			ORDER BY note_id DESC
+		";
+
+		$per_page = isset( $args['per_page'] ) ? intval( $args['per_page'] ) : 10;
+		if ( $per_page <= 0 ) {
+			$per_page = 1;
```

Comment:
Again should this be 10 here?

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/507/commits/931b45f71d97a3ea5ee15244c68355514e947d95